app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+  before_action :set_locale
+  private
+  def set_locale
```

Comment:
```suggestion
  before_action :set_locale
  
  private
  
  def set_locale
```
Thêm line trống giữa các thành phần để dễ phân biệt